src/components/builder/values/values-config.tsx

```diff
@@ -1,11 +1,26 @@
 import {useFormikContext} from 'formik';
 import {useLayoutEffect} from 'react';
 
+import {hasOwnProperty} from '@/types';
+
 import ItemsExpression from './items-expression';
 import {SchemaWithDataSrc} from './types';
 import ValuesSrc from './values-src';
 import ValuesTable, {ValuesTableProps} from './values-table';
 
+function isNestedKeySet(obj: {}, path: string): boolean {
```

Comment:
```suggestion
/**
 * Check if the dotted `path` exists on `obj`.
 * @example
 * ```
 * isNestedKeySet({my: {path: true}}, 'my.path') // true
 * ```
 */
function isNestedKeySet(obj: {}, path: string): boolean {
```